timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -326,11 +326,14 @@ public void timestampServiceShouldRespectDistinctClientsWhenIssuingTimestamps()
         long firstServiceFirstTimestamp = timestampService1.getFreshTimestamp();
         long secondServiceFirstTimestamp = timestampService2.getFreshTimestamp();
 
-        long firstServiceSecondTimestamp = timestampService1.getFreshTimestamp();
+        long firstServiceSecondTimestamp = timestampService1
+                .getFreshTimestamps(100)
+                .getUpperBound();
+
         long secondServiceSecondTimestamp = timestampService2.getFreshTimestamp();
 
-        assertEquals(firstServiceFirstTimestamp + 1, firstServiceSecondTimestamp);
-        assertEquals(secondServiceFirstTimestamp + 1, secondServiceSecondTimestamp);
+        assertThat(firstServiceSecondTimestamp - firstServiceFirstTimestamp).isGreaterThanOrEqualTo(100L);
```

Comment:
Technically speaking this can fail because getFreshTimestamps might not return the full 100 timestamps (the API doesn't guarantee that). Maybe this is too much but we could do something like

```
TimestampRange tr = timestampService1.getFreshTimestamps(100);
while (tr.size() != 100) {
  tr = timestampService1.getFreshTimestamps(100);
}
long firstServiceSecondTimestamp = tr.getUpperBound();
```

Reply:
that is right. Used the already existing `getFortyTwoFreshTimestamps()` instead, that should address the problem.